doc/source/contributing_code.rst

```diff
@@ -204,49 +210,76 @@ This sets the number of files to 4096. However, this command only applies to you
 Building MDAnalysis
 -------------------
 
-With conda
-----------
+With mamba/conda
+----------------
 
 .. note::
     Make sure that you have :ref:`cloned the repository <forking-code-repo>`
-    and activated your virtual environment with :code:`conda activate mdanalysis-dev`.
+    and activated your virtual environment with :code:`mamba activate mdanalysis-dev`.
 
-First we need to install dependencies. You'll need a mix of conda and pip installations:
+First we need to install the dependencies. To install the base MDAnalysis
+dependencies, do the following:
 
     .. code-block:: bash
 
-        conda install -c conda-forge \
-          'Cython>=0.28' \
-          'numpy>=1.21.0' \
-          'biopython>=1.80' \
-          'networkx>=2.0' \
-          'GridDataFormats>=0.4.0' \
+        mamba install -c conda-forge \
+          'cython>=0.28' \
+          'fasteners' \
+          'griddataformats>=0.4.0' \
+          'hypothesis' \
+          'matplotlib-base>=1.5.1' \
+          'mdahole2-base' \
+          'mda-xdrlib' \
           'mmtf-python>=1.0.0' \
-          'joblib>=0.12' \
+          'numpy>=1.23.2' \
+          'packaging' \
+          'pathsimanalysis' \
+          'pytest' \
           'scipy>=1.5.0' \
-          'matplotlib>=1.5.1' \
+          'threadpoolctl' \
           'tqdm>=4.43.0' \
-          'threadpoolctl'\
-          'packaging' \
-          'fasteners' \
-          'netCDF4>=1.0' \
-          'h5py>=2.10' \
+          'waterdynamics'
+
+
+You can also install the following optional dependencies, although please note
+that they many not all be available for your machine type. Specifically;
+hole2, and distopia are only available on linux + x86_64 machines, and openmm
+is not available for windows.
```

Comment:
Maybe add "Remove the packages listed as incompatible from the command below and run it again."

Alternatively, we can show example code for different os with `sphinxcontrib.osexample` (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html)

Reply:
`sphinxcontrib.osexample` as used in https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html#code-examples-in-multiple-languages looks like a neat addition. Maybe open new issue?